backend/src/test/java/capstone/kpiboard/KpiControllerIntegrationTest.java

```diff
@@ -107,4 +113,39 @@ void deleteKpiByIdTestKpiDoesntExist() throws Exception {
         Assertions.assertTrue(errorMessage.contains("timestamp"));
         Assertions.assertTrue(errorMessage.contains("message"));
     }
+
+    @Test
+    @DirtiesContext
+    void updateKpiById() throws Exception {
+
+        String result = mockMvc.perform(post("/api/kpis")
```

Comment:
Nur als Alternative: um objectMapper zu vermeiden kannst du ID-Generierung in Integration-test mocken (mit @MockBean) und dann einfach als string "123" im Test nutzen